test/framework/src/test/java/org/elasticsearch/test/test/LoggingListenerTests.java

```diff
@@ -33,6 +33,17 @@
 import static org.hamcrest.CoreMatchers.nullValue;
 
 public class LoggingListenerTests extends ESTestCase {
+
+    public void testTestRunStartedSupportsClassInDefaultPackage() throws Exception {
+        LoggingListener loggingListener = new LoggingListener();
+        Description description = Description.createTestDescription(Class.forName("Dummy"), "dummy");
+        try {
+            loggingListener.testRunStarted(description);
+        } catch (Exception e) {
```

Comment:
There's no need to catch and fail here, an uncaught exception thrown by `loggingListener.testRunStarted` will cause the test to fail already, catching and calling fail adds nothing and even swallows the thrown exception.